### DISCUSSION TITLE
accessToken过期是自动刷新吗？

### DISCUSSION THREAD

**Discussion by lshfong** [w7corp/easywechat#2783] [General]
accessToken过期是自动刷新吗？还是需要自己判断然后刷新
`$app = Factory::miniProgram($config);
        $accessToken = $app->access_token;
        $token = $accessToken->getToken();`
我这样写，但是有时候会出现accessToken过期的错误。

**Comment by iluyuns** [comment]
是自动刷新，推测你在开发环境 也 运行了程序获取了 Token ,导致了线上失效。 开发环境 配置一个接口获取 线上的 token,/ 或者统一一个 线上线下都能访问的 ew 缓存
